src/cmd_deploy.rs

```diff
@@ -25,8 +25,23 @@ pub fn command_deploy< 'a >( matches: &clap::ArgMatches< 'a > ) -> Result< (), E
     let target = targets[ 0 ];
     let result = project.build( &config, target )?;
 
-    let deployment = Deployment::new( package, target, &result )?;
-    let directory = project.target_directory().join( "deploy" );
+    let target_config = match project.config_of_default_target() {
+        Some(config) => config.clone(),
+        None => ::config::PerTargetConfig::default()
+    };
```

Comment:
Instead of this please add `deploy_path` and `serve_path` getters to `Project`. Have them always return `String`.

Reply:
I agree with the `deploy_path`. But with `serve_path`, I think return `Option<String>` is better, because we do different things if there is a `serve-path` or not.

Reply:
How do we do different things if there is a `serve-path`? By default the behavior should be the same as if `serve-path` was set to `/`, no?